src/common/snippets/include/snippets/lowered/pass/pass.hpp

```diff
@@ -91,7 +91,7 @@ class ConstPass : public PassBase {
  */
 class RangedPass : public PassBase {
 public:
-    OPENVINO_RTTI("snippets::lowered::pass::RangedPass", "", PassBase)
+    OPENVINO_RTTI("snippets::lowered::pass::RangedPass")
```

Comment:
@a-sidorova @IvanNovoselov Was it accidentally reverted while merging upstream/master? It was intentionally added in #28394.